scripts/preloadHTTPSE.js

```diff
@@ -1,12 +1,11 @@
 'use strict'
 const fs = require('fs')
 const http = require('https')
-const parseString = require('xml2js').parseString
 const levelup = require('level')
 const rmDir = require('./util').rmDir
 const exec = require('child_process').exec
 
-const xpiVersion = '5.2.21' // Manually update this to latest version
+const xpiVersion = '2017.12.6' // Manually update this to latest version
```

Comment:
please also bump the version in `package.json` to `6.0.0`. this creates a new ruleset download in S3 instead of overwriting the current one (safer in case the new one has breaking changes).